packages/autorest.typescript/src/generators/static/packageFileGenerator.ts

```diff
@@ -189,6 +182,19 @@ function regularAutorestPackage(
     packageInfo.homepage = `https://github.com/Azure/azure-sdk-for-js/tree/main/${azureOutputDirectory}`;
   }
 
+  if (azureSdkForJs) {
+    packageInfo.devDependencies["@azure/dev-tool"] = "^1.0.0";
+    packageInfo.scripts["build"] =
+      "npm run clean && tsc && dev-tool run bundle && npm run minify && mkdirp ./review && npm run extract-api";
+  } else {
+    packageInfo.devDependencies["@rollup/plugin-commonjs"] = "^24.0.0";
+    packageInfo.devDependencies["@rollup/plugin-json"] = "^6.0.0";
+    packageInfo.devDependencies["@rollup/plugin-multi-entry"] = "^6.0.0";
+    packageInfo.devDependencies["@rollup/plugin-node-resolve"] = "^13.1.3";
+    packageInfo.devDependencies["rollup"] = "^2.66.1";
+    packageInfo.devDependencies["rollup-plugin-sourcemaps"] = "^0.6.3";
+  }
```

Comment:
Thanks for contributing this! Could you help understand below questions?

- Does this apply for both RLC, HLC and Modular?
- Doest this change apply for both swagger and typespec path?

If yes I think we still have changes in rlc-common and typespec-ts.

Reply:
Yes you are right,  any packages in the js repo should use dev-tool to bundle.

Reply:
Turns out that you and Qiaoqiao already use `dev-tool run bundle` for js repo in rlc-common and typespec-ts :)